jquery.unobtrusive-ajax.js

```diff
@@ -1,18 +1,16 @@
-/* 
- * Microsoft grants you the right to use these script files for the sole
- * purpose of either: (i) interacting through your browser with the Microsoft
- * website or online service, subject to the applicable licensing or use
- * terms; or (ii) using the files as included with a Microsoft product subject
- * to that product's license terms. Microsoft reserves all other rights to the
- * files not expressly granted by Microsoft, whether by implication, estoppel
- * or otherwise. Insofar as a script file is dual licensed under GPL,
- * Microsoft neither took the code under GPL nor distributes it thereunder but
- * under the terms set out in this paragraph. All notices and licenses
- * below are for informational purposes only.
-/*!
-** Unobtrusive Ajax support library for jQuery
-** Copyright (C) Microsoft Corporation. All rights reserved.
-*/
+// Microsoft grants you the right to use these script files for the sole
+// purpose of either: (i) interacting through your browser with the Microsoft
+// website or online service, subject to the applicable licensing or use
+// terms; or (ii) using the files as included with a Microsoft product subject
+// to that product's license terms. Microsoft reserves all other rights to the
+// files not expressly granted by Microsoft, whether by implication, estoppel
+// or otherwise. Insofar as a script file is dual licensed under GPL,
+// Microsoft neither took the code under GPL nor distributes it thereunder but
+// under the terms set out in this paragraph. All notices and licenses
+// below are for informational purposes only.
+// Unobtrusive Ajax support library for jQuery
+// Copyright (C) Microsoft Corporation. All rights reserved.
```

Comment:
Can we change the order of things here? I suggest something like:

```
// <title text>
// <copyright text>
// <newline>
// <license ...>
// <license ...>
// <license ...>
// <license ...>
```

That might look nicer than the current order. Especially considering we don't use that NuGet license header thing anyway.